system_files/silverblue/etc/dconf/db/distro.d/02-bluefin-keybindings

```diff
@@ -3,25 +3,15 @@
 
 [org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0]
 binding='<Control><Alt>t'
-command='/usr/libexec/distrobox-quadlet-ptyxis.sh Host'
-name='Host Ptyxis'
+command='/usr/bin/ptyxis --tab-with-profile="2871e8027773ae74d6c87a5f659bbc74" --new-window'
```

Comment:
The profile was just so that we knew for sure we were opening a ptyxis host terminal instead of some random container the user might have. I personally dont really think that makes sense anymore, though

Reply:
Yeah I'm thinking we should try to go as vanilla as possible from now on, but we shouldn't surprise existing users, are there any downsides you can think of?

Reply:
Not really! We would just rely on the default that ptyxis does instead of forcing host. That may be weird if someone still has something like the bluefin-cli container as their default, but otherwise it should work fine.

Reply:
So, what do you think about removing this along with other ptyxis manipulation stuff on another PR?